sdks/java/harness/src/main/java/org/apache/beam/fn/harness/status/BeamFnStatusClient.java

```diff
@@ -58,6 +59,7 @@ public class BeamFnStatusClient implements AutoCloseable {
   private final MemoryMonitor memoryMonitor;
   private final Cache<?, ?> cache;
 
+  @SuppressFBWarnings("SC_START_IN_CTOR") // for memory monitor thread
```

Comment:
not sure why this popped up only now, the thread start was there previously too.

Reply:
The check might have some cache mechanism and only check recently changed paths